src/SingleBookingPage/NearestBooking.js

```diff
@@ -122,7 +122,13 @@ class NearestBooking extends React.Component<Props> {
           <Notification isUrgent={isUrgent} hoursLeft={deltaInHours} />
           {this.renderByType(nearestBooking.booking)}
           <div className="buttons">
-            <button className="manage-booking">Manage my booking</button>
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href={nearestBooking.booking.directAccessURL}
+            >
+              <button className="manage-booking">Manage my booking</button>
```

Comment:
what do you think about adding `cursor: pointer;` to the class? so it feels like is clickable

Reply:
Yes, agreed! How do you do this changes, in the same branch? Thanks @jalbertsr

Reply:
Yep, change it in the same branch 🙂

Reply:
Done! 🙂